packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -59,14 +59,20 @@ const createProjectPrompt = async (
   skipTemplatePrompt = false
 ) => {
   let projectTemplates = [];
+  let selectedTemplate;
```

Comment:
The `hs project create` command wasn't using the `--template` option correctly. That option takes in a string, but all of the following utils are expecting an object that has a shape like [this](https://github.com/HubSpot/hubspot-project-components/blob/main/config.json#L10-L15). Now we'll check to see if the template option matches any of the available templates in the repo's config.